rplugin/python3/defx/source/file/__init__.py

```diff
@@ -60,6 +60,8 @@ def gather_candidates(
                     'is_directory': safe_call(entry.is_dir, False),
                     'action__path': entry,
                 })
+            if context.show_parent:
+                candidates.append({'word':'../','is_directory':True, 'action__path':path.resolve().parent,})
```

Comment:
`candidates.append({'word' :'../', 'is_directory': True, 'action__path': path.resolve().parent})`

Reply:
And please fix the lint.